internal/refactoring/move_statement.go

```diff
@@ -86,11 +86,20 @@ func findMoveStatements(cfg *configs.Config, into []MoveStatement) []MoveStateme
 // We should think very hard before adding any _new_ implication rules for
 // moved statements.
 func ImpliedMoveStatements(rootCfg *configs.Config, prevRunState *states.State, explicitStmts []MoveStatement) []MoveStatement {
-	return impliedMoveStatements(rootCfg, prevRunState, explicitStmts, nil)
+	return impliedMoveStatements(rootCfg, prevRunState, explicitStmts)
```

Comment:
We can merge the public/private function here as you have untangled the additional parameter.